src/main/java/com/fundation/logic/view/PlayerJFrame.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.fundation.logic.view;
+
+import com.sun.jna.NativeLibrary;
+import uk.co.caprica.vlcj.component.EmbeddedMediaPlayerComponent;
+import uk.co.caprica.vlcj.runtime.RuntimeUtil;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JSlider;
+import javax.swing.ImageIcon;
+import javax.swing.Box;
+import javax.swing.WindowConstants;
+import java.awt.Rectangle;
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Dimension;
+import java.io.File;
+
+/**
+ * Implements the player frame with playing control buttons
+ *
+ * @author Melissa Román
+ * @version 1.0
+ */
+class PlayerJFrame extends JFrame {
+
+    private JLabel playButton = new JLabel();
+    private JLabel stopButton = new JLabel();
+    private JLabel pauseButton = new JLabel();
+    private int vol = 50;
+    private JPanel headerPanel = new JPanel();
+    private JPanel playingPanel = new JPanel();
+    private JPanel bottomPanel = new JPanel();
```

Comment:
Please separate all declared object of the initialize objects

Reply:
Done